terraform/api_gateway.tf

```diff
@@ -46,7 +46,7 @@ module "query_get" {
   authorizer_id = aws_api_gateway_authorizer.api_key.id
   api_resource = module.query_resource.aws_api_gateway_resource
 
-  require_api_key = true
+  require_api_key = false
```

Comment:
Will need to disable api keys in the backend by commenting out [this](https://github.com/wri/gfw-data-api/blob/2eb041b99ed3af0ac9ad7c954de50561c3650391/app/routes/datasets/queries.py#L124) and the remaining endpoints in the module as well.